src/server/middleware/proxyMiddleware.ts

```diff
@@ -56,13 +56,22 @@ const staticAssetsMiddleware = createProxyMiddleware('/static', {
   target: 'http://localhost:8081'
 });
 
-const apiProxyMiddleware = createProxyMiddleware('/api', {
+const apiProxyMiddleware = createProxyMiddleware(['/api/**', '!/api/docs/**'], {
   target: 'https://staging-2020.ensembl.org',
   changeOrigin: true,
   secure: false
 });
 
-let proxyMiddleware = [apiProxyMiddleware];
+const docsProxyMiddleware = createProxyMiddleware('/api/docs/**', {
+  target: 'http://localhost:3000',
```

Comment:
was this added to test locally?  we have api/docs/** mentioned in the above apoProxyMiddleware.

Reply:
Yes, thank you! This shouldn't have gone in the PR.